src/bitte_plugin/src/handler/open_api_handler.rs

```diff
@@ -119,6 +122,16 @@ pub async fn open_api_specification() -> Json<Value> {
                     },
                     "example": "@ryan_soury",
                     "description": "The tweet account to notify when is reward/post is complete"
+                  },
+                  {
+                    "in": "query",
+                    "name": "computed_cost",
```

Comment:
**Note:** There is a view function on the Contract that can send metrics to a reader to provide estimated cost.

If this is the case, why are we providing this as a parameter?
This is because the LLM operates as a chain of operations.
Therefore, the output of the prior LLM output is used as an input.

**Note 2:** We're adding 20% to this to avoid situation where submits, and then the metrics have changed.